tasks/eth/fp-recovery/001-blacklist-game/.env

```diff
@@ -0,0 +1,5 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+SUPERCHAIN_CONFIG_ADDR=0x95703e0982140D16f8ebA6d158FccEde42f04a4C
```

Comment:
Looks like these are in each `.env` file but are unused

Reply:
Removed unused `SUPERCHAIN_CONFIG_ADDR`